### DISCUSSION TITLE
Pinokio - Live Portrait update frozen

### DISCUSSION THREAD

**Discussion by PBandJ-Dev** [KlingAIResearch/LivePortrait#334] [Q&A]
Hey guys need some help updating - seems to be stuck at 

C:\pinokio\api\liveportrait.git>conda_hook && conda deactivate && conda deactivate && conda deactivate && conda activate base && conda install -y -c conda-forge 7zip && 7z x installer.zip && move LivePortrait-Windows-v20240806 app && del installer.zip
Channels:
 - conda-forge
 - defaults
 - nvidia/label/cuda-12.1.0
Platform: win-64
Collecting package metadata (repodata.json): done
Solving environment: done

# All requested packages already installed.

7-Zip 23.01 (x64) : Copyright (c) 1999-2023 Igor Pavlov : 2023-06-20

Scanning the drive for archives:
1 file, 5235950042 bytes (4994 MiB)

Extracting archive: installer.zip
--
Path = installer.zip
Type = zip
Physical Size = 5235950042
64-bit = +
Characteristics = Zip64

    
Would you like to replace the existing file:
  Path:     .\LivePortrait-Windows-v20240806\.gitignore
  Size:     697 bytes (1 KiB)
  Modified: 2024-08-04 16:55:13
with the file from archive:
  Path:     LivePortrait-Windows-v20240806\.gitignore
  Size:     697 bytes (1 KiB)
  Modified: 2024-08-04 16:55:13
? (Y)es / (N)o / (A)lways / (S)kip all / A(u)to rename all / (Q)uit? 

It wont let me enter Y or anything else. its just frozen - any ideas?

**Comment by githubuser927** [comment]
Exactly the same problem here. No idea how to fix. Tried uninstalling, reinstalling, nothing seems to work.
